conf4j-spring/src/main/resources/META-INF/com/sabre/oss/conf4j/conf4j.xsd

```diff
@@ -172,4 +171,72 @@
             </xsd:complexContent>
         </xsd:complexType>
     </xsd:element>
+
+    <xsd:element name="converter">
+        <xsd:annotation>
+            <xsd:documentation>
+                Registers a type converter to the context.
+            </xsd:documentation>
+        </xsd:annotation>
+        <xsd:complexType>
+            <xsd:complexContent>
+                <xsd:extension base="beans:identifiedType">
+                    <xsd:attribute name="order" type="xsd:int">
+                        <xsd:annotation>
+                            <xsd:documentation>
+                                The order of the converter.
+                            </xsd:documentation>
+                        </xsd:annotation>
+                    </xsd:attribute>
+                    <xsd:attribute name="class" type="xsd:string">
+                        <xsd:annotation>
+                            <xsd:documentation>
+                                The fully qualified name of the converter type class.
+                            </xsd:documentation>
+                            <xsd:appinfo>
+                                <tool:annotation>
+                                    <tool:expected-type type="java.lang.Class"/>
+                                    <tool:exports type="com.sabre.oss.conf4j.converter.TypeConverter"/>
+                                </tool:annotation>
+                            </xsd:appinfo>
+                        </xsd:annotation>
+                    </xsd:attribute>
+                </xsd:extension>
+            </xsd:complexContent>
+        </xsd:complexType>
+    </xsd:element>
+
+    <xsd:element name="delegatingConverterFactory">
```

Comment:
Spring default convention is to use dashes to separate words instead of camel case. I would also suggest finding better name (as well as for `DelegatinConverterFactory` class).